pipeline-templates/stage/build.yml

```diff
@@ -0,0 +1,37 @@
+stages:
+- stage: build
+  displayName: Build Image
+  jobs:
+  - job: build_image
+    displayName: Build Image
+    pool:
+      name: "DAS - Continuous Integration"
+      demands: Agent.OS -equals Linux
+    steps:
+      - task: gittools.gitversion.gitversion-task.GitVersion@5
+        displayName: GitVersion
+        inputs:
+          configFilePath: GitVersion.yml
+          updateAssemblyInfo: true
+
+      - task: Docker@2
+        displayName: Login to Docker Hub
+        inputs:
+          command: login
+          containerRegistry: "Apprenticeships DevOps Docker Hub"
+
+      - task: Docker@2
+        displayName: Build Image
+        inputs:
+          command: build
+          dockerfile: Dockerfile
+          repository: apprenticeshipsdevops/das-status-cake-exporter
+          tags: |

```

Comment:
Is the | needed if there is just one line for the tag?

Reply:
Removed